src/cli/cli.js

```diff
@@ -96,7 +96,9 @@ async function runTests (argParser) {
         .screenshots(opts.screenshots, opts.screenshotsOnFails, opts.screenshotPathPattern)
         .startApp(opts.app, opts.appInitDelay);
 
-    runner.once('done-bootstrapping', () => log.hideSpinner());
+    runner.initEvents({
```

Comment:
What about raising `done-bootstrapping` event in the `Runner._createRunnableConfiguration` method?
```js
_createRunnableConfiguration () {
        return this.bootstrapper
            .createRunnableConfiguration()
            .then(runnableConfiguration => {
                this.emit('done-bootstrapping');

                return runnableConfiguration;
            });
    }

```